test/dotnet/input/Storedproc/TestStoredProcedure.txt

```diff
@@ -271,4 +271,12 @@ storedproc#!#prep#!#sp_test25#!#int|-|a|-|20|-|input#!#int|-|b|-|10|-|output#!#i
 storedproc#!#prep#!#sp_test25#!#int|-|a|-|20|-|input#!#int|-|b|-|10|-|inputoutput#!#int|-|c|-|10|-|input#!#int|-|d|-|10|-|output
 
 DROP PROCEDURE sp_test25
-# Test (24): End
\ No newline at end of file
+# Test (24): End
+
+# Test (25): Test sys.sp_reset_connection stored procedure
+CREATE TABLE #babel_temp_table (ID INT identity(1,1), Data INT)
+INSERT INTO #babel_temp_table (Data) VALUES (100), (200), (300)
+SELECT * from #babel_temp_table
+storedproc#!#prep#!#sys.sp_reset_connection#!#
+SELECT * from #babel_temp_table
```

Comment:
This is not actually testing the RPC in this connection, can we also have an RPC after storedproc?